alpha/scripts/utils/updatePartnerEntries2Sphinx.php

```diff
@@ -51,8 +51,16 @@
 		if ($mode == 'execute')
 		{
 			echo "Updating [". $item->getId() ."] ... "; 
-			$sphinx->saveToSphinx($item, false, true);
-			echo "Done\n";
+			try {
+				$sphinx->saveToSphinx($item, false, true);
+				echo 'Done' . PHP_EOL;
+			} catch (PDOException $e) {
+				if (strpos($e->getMessage(), 'unknown column') !== false) {
+					echo '[unknown column] error occurred while updating [' . $item->getId() . ']: ' . $e->getMessage() . PHP_EOL;
```

Comment:
I think we can take the opportunity and change all 'echo' to KalturaLog
The script was written in 2011, KalturaLog was added at 2013 iirc